src/sentry/static/sentry/app/views/issueList/createSavedSearchButton.tsx

```diff
@@ -80,12 +89,12 @@ class CreateSavedSearchButton extends React.Component {
     }
   };
 
-  handleChangeName = val => {
-    this.setState({name: val});
+  handleChangeName = (val: string | number | boolean) => {
```

Comment:
```suggestion
  handleChangeName = (val: Parameters<NonNullable<TextField['props']['onChange']>>[0]) => {
```

Reply:
If you go down the `Parameters` route you might want to define a type for it so you're not repeating this expression.